opsdroid/tests/test_constraints.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+import asynctest.mock as amock
+
+from opsdroid.cli.start import configure_lang
+from opsdroid.core import OpsDroid
+from opsdroid.events import Message
+from opsdroid.matchers import match_regex
+from opsdroid import constraints
+
+configure_lang({})  # Required for our internationalization of error messages
+
+
+@pytest.fixture
+def opsdroid():
+    """An instance of the OpsDroid class."""
+    with OpsDroid() as opsdroid:
+        yield opsdroid
```

Comment:
This fixture already exists so we don't need this.

https://github.com/opsdroid/opsdroid/blob/a8bddfd0c0361e6e5d25ee65fb00d00c30881705/opsdroid/testing/fixtures.py#L68